### PR TITLE
JSON generic feed

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21755

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by teizenman** [comment]
Docs-approved from https://github.com/demisto/content/pull/5942

File: Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py [hunk 1]
```diff
@@ -154,6 +184,32 @@ def fetch_indicators_command(client: Client, indicator_type: str, **kwargs) -> U
     return indicators
 
 
+def extract_all_fields_from_indicator(indicator, indicator_key):
```
Comment (ronykoz) [inline Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py]: Why not to join the values here? you may override keys here

**Comment by ronykoz** [comment]
And you have conflicts

File: Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py (hunk 1 above)
Comment (teizenman) [reply Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py]: Fixed

**Pull request merged**